lib/modules/manager/nuget/update.spec.ts

```diff
@@ -7,6 +8,14 @@ const minimumContent =
   '<Project Sdk="Microsoft.NET.Sdk"><PropertyGroup><Version>1</Version></PropertyGroup></Project>';
 const prereleaseContent =
   '<Project Sdk="Microsoft.NET.Sdk"><PropertyGroup><Version>1.0.0-1</Version></PropertyGroup></Project>';
+const prefixContent =
```

Comment:
inline variables which are only used once for better nodejs garbage collection

Reply:
`prereleaseContent` is also only used in one test—should I fix that while I'm here?

Reply:
sure